Frontend/src/pages/SystemDetails.js

```diff
@@ -8,14 +16,13 @@ import CriticalPointsTable from '../components/FunctionDetail/CriticalPointsTabl
 import CriticalPointPortraitTable from '../components/FunctionDetail/CriticalPointPortraitTable'
 import ModelsTable from '../components/FunctionDetail/ModelsTable'
 import CitationsTable from '../components/FunctionDetail/CitationsTable'
-import { useFilters } from '../context/FilterContext';
 
 function SystemDetails() {
     const [data, setData] = useState({});
     const label = useParams().label;
     useEffect(() => {
       fetchDataForCSV();
-  },[]); 
+  }); 
```

Comment:
I don't think we should remove the empty array, can you check oncE?

Reply:
I can put it back in, we just get a linting warning then.